pkgs/development/python-modules/funsor/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, makefun
+, multipledispatch
+, numpy
+, opt-einsum
+, typing-extensions
+, pyro-ppl
+, torch
+, pandas
+, pillow
+, pyro-api
+, pytest-xdist
+, pytestCheckHook
+, requests
+, scipy
+, torchvision
+}:
+
+let
+  self = buildPythonPackage rec {
+    pname = "funsor";
+    version = "0.4.5";
+
+    disabled = pythonOlder "3.7";
+
+    src = fetchFromGitHub {
+      owner = "pyro-ppl";
+      repo = "funsor";
```

Comment:
```suggestion
      repo = pname;
```

Reply:
This practice has been discouraged by maintainers recently to avoid errors if the package gets renamed.

Reply:
fair enough